### PR TITLE
Added option to tune Jersey Client timeouts for some tests.

### PR DESCRIPTION
**Pull request by harnash**
It seemed that on POZ-DEV we were suffering from timouts on some
integration tests. This change allows to set connection timeouts for a
specific test/cases.

@Wikia/services-team

### CODE REVIEW

**Comment by WikiaTech** [comment]
SonarQube analysis reported 2 issues:
* MAJOR 2 major

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MAJOR [1 duplicated blocks of code must be removed.](https://github.com/Wikia/pandora/blob/42e4d2c5d82f17ac57fc0039ae7e3398aa8d12e9/service/user-preference/src/test/java/com/wikia/userpreference/resources/UserPreferenceLegacyResourceIntegrationTest.java) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=common-java%3ADuplicatedBlocks)
* MAJOR [1 duplicated blocks of code must be removed.](https://github.com/Wikia/pandora/blob/42e4d2c5d82f17ac57fc0039ae7e3398aa8d12e9/service/user-preference/src/test/java/com/wikia/userpreference/resources/UserPreferenceResourceIntegrationTest.java) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=common-java%3ADuplicatedBlocks)

**Comment by harnash** [comment]
It seems that tests are unreliable again...

File: service/user-preference/src/test/java/com/wikia/userpreference/helpers/IntegrationTestHelper.java [hunk 1]
```diff
@@ -11,9 +11,13 @@ private IntegrationTestHelper() {
   }
 
   public static JerseyClientConfiguration getSaneJerseyClientConfiguration() {
```
Comment (sqreek) [inline service/user-preference/src/test/java/com/wikia/userpreference/helpers/IntegrationTestHelper.java]: can we get rid of this?

**Comment by harnash** [comment]
@sqreek: done

**Comment by WikiaTech** [comment]
SonarQube analysis reported 2 issues:
* MAJOR 2 major

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MAJOR [1 duplicated blocks of code must be removed.](https://github.com/Wikia/pandora/blob/cfc5a95a4fb6f76a7183173fc747502e46003f34/service/user-preference/src/test/java/com/wikia/userpreference/resources/UserPreferenceLegacyResourceIntegrationTest.java) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=common-java%3ADuplicatedBlocks)
* MAJOR [1 duplicated blocks of code must be removed.](https://github.com/Wikia/pandora/blob/cfc5a95a4fb6f76a7183173fc747502e46003f34/service/user-preference/src/test/java/com/wikia/userpreference/resources/UserPreferenceResourceIntegrationTest.java) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=common-java%3ADuplicatedBlocks)

**Comment by drsnyder** [comment]
Do we know that this was the cause of the failure?

**Comment by harnash** [comment]
Unfortunately no. It seems that initiating connection and first query is slower than on SJC-DEV cluster. I will try to dig deeper but had no time for that now.

**Comment by sqreek** [comment]
ping @harnash 
is this is still valid?

**Comment by harnash** [comment]
I think it is heavily outdated and this problem hasn't appeared since then. I think that having own Mesos cluster helped to solve that problem. Closing.

**Comment by WikiaTech** [comment]
SonarQube analysis reported 2 issues:
* MAJOR 2 major

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MAJOR [1 duplicated blocks of code must be removed.](https://github.com/Wikia/pandora/blob/cfc5a95a4fb6f76a7183173fc747502e46003f34/service/user-preference/src/test/java/com/wikia/userpreference/resources/UserPreferenceLegacyResourceIntegrationTest.java) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=common-java%3ADuplicatedBlocks)
* MAJOR [1 duplicated blocks of code must be removed.](https://github.com/Wikia/pandora/blob/cfc5a95a4fb6f76a7183173fc747502e46003f34/service/user-preference/src/test/java/com/wikia/userpreference/resources/UserPreferenceResourceIntegrationTest.java) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=common-java%3ADuplicatedBlocks)

**Pull request closed without merging**